site/docs/plugins/entity-parser.md

```diff
@@ -0,0 +1,237 @@
+| `strikethrough`         | `CommonEntity`      | `<del class="tg-strikethrough"> ... </del>`                                                                                                                                        |
+| `text_link`             | `TextLinkEntity`    | `<a class="tg-text-link" href="${options.entity.url}"> ... </a>`                                                                                                                   |
+| `text_mention`          | `TextMentionEntity` | `<a class="tg-text-mention" href="https://t.me/${options.entity.user.username}"> ... </a>` or `<a class="tg-text-mention" href="tg://user?id=${options.entity.user.id}"> ... </a>` |
+| `underline`             | `CommonEntity`      | `<span class="tg-bot-command"> ... </span>`                                                                                                                                        |
+| `url`                   | `CommonEntity`      | `<a class="tg-url" href="${options.text}"> ... </a>`                                                                                                                               |
+
+If you are unsure which interface is correct, refer to how the [Renderer](https://github.com/quadratz/telegram-entities-parser/blob/main/src/renderers/renderer.ts) or [RendererHtml](https://github.com/quadratz/telegram-entities-parser/blob/main/src/renderers/renderer_html.ts) is implemented.
+
+### Customize the Text Sanitizer
+
+The output text is sanitized by default to ensure proper HTML rendering and prevent XSS vulnerabilities.
+
+| Input | Output   |
+| ----- | -------- |
+| `&`   | `&amp;`  |
+| `<`   | `&lt;`   |
+| `>`   | `&gt;`   |
+| `"`   | `&quot;` |
+| `'`   | `&#x27;` |
+
+For example, the result `<b>Bold</b> & <i>Italic</i>` will be sanitized to `<b>Bold</b> &amp; <i>Italic</i>`.
+
+You can override this behavior by specifying a `textSanitizer` when instantiating the [`EntitiesParser`](https://github.com/quadratz/telegram-entities-parser/blob/main/src/mod.ts):
+
+- If you do not specify `textSanitizer`, it will default to using [`sanitizerHtml`](https://github.com/quadratz/telegram-entities-parser/blob/main/src/utils/sanitizer_html.ts) as the sanitizer.
+- Setting the value to `false` will skip sanitization, keeping the output text as the original.
+  This is not recommended, as it may result in incorrect rendering and make your application vulnerable to XSS attacks.
+  Ensure proper handling if you choose this option.
+- If you provide a function, it will be used instead of the default sanitizer.
+
+```ts
+const myTextSanitizer: TextSanitizer = (options: TextSanitizerOption): string =>
+  // Replace dangerous character
+  options.text.replace(/[&<>"']/g, (match) => {
```

Comment:
Is there a reason to avoid `replaceAll`?

Reply:
No particular reason. The global flag `g` in the regex accomplishes the same thing.

Reply:
I'm aware, `replaceAll` was introduced because the `g` flag is poorly understood by many developers, and it can easily be forgotten. But anyway, I wasn't saying that the code isn't correct, so we can keep it that way, if you prefer.

Reply:
I find `replaceAll` to be more readable, so I’ll switch to using it.

Reply:
```suggestion
  options.text.replaceAll(/[&<>"']/, (match) => {
```